src/libs/lib.c

```diff
@@ -1457,6 +1457,10 @@ void dt_lib_cancel_postponed_update(dt_lib_module_t *mod)
   }
 }
 
+gboolean dt_lib_presets_can_autoapply(dt_lib_module_t *mod)
+{
+  return (mod->preset_autoapply && mod->preset_autoapply(mod));
```

Comment:
I would add a default cb for preset_autoapply which always return FALSE and only do here:

`return mod->preset_autoapply(mod);`

This is what is done in many places IIRC.

Reply:
See default_expandable for example. BTW, this routine should probably also return a gboolean. I'll fix.